### ISSUE TITLE
[Bug]: is coordination matrix correct? 

### ISSUE THREAD

**Issue by onderilkesever** [ThatOpen/engine_web-ifc#1068]
### What happened?

Hi! 
I am seeing something weird with `COORDINATE_TO_ORIGIN` property and `coordinationMatrix`. I basically have two scenarios that i would like to compare to show you where my confusion arises. 

1. 1st scenario, `COORDINATE_TO_ORIGIN = false` -> I load my model, and center of the bounding box of my model is at below coordinates. 
![Screenshot 2024-10-10 at 10 22 51](https://github.com/user-attachments/assets/9d87fa4c-3440-442f-a0d9-7aea1dfb6355)
2. 2nd scenario, `COORDINATE_TO_ORIGIN = true` -> I load my model similarly, and center of the bounding box of my model is at below coordinates. 
![Screenshot 2024-10-10 at 10 28 40](https://github.com/user-attachments/assets/c80c47f6-bb77-4341-a1a3-b4f0461fb2d6)
So far looks correct as the model has been brought to center. And we have `coordinationMatrix` property which holds the translation that has been applied to model as far as i know. So ideally, I believe if we apply the inverse of `coordinationMatrix`, we should expect model to be at the same place as 1st scenario where `COORDINATE_TO_ORIGIN = false`. However, if you look at the inverse of  `coordinationMatrix`, i see below.  
[
    1,    0,    0,    0,
    0,    1,    0,    0,
    0,    0,    1,    0,
    689674.9375,    -5335900,    13.474897384643555,    1
]

Basically if you apply this transformation, you would see Y axis is not matching in both scenarios. 
I wonder is this a bug or am i missing something?
(PLEASE NOTE THAT MY APPLICATION USES Z UP, THATS WHY ALL THE Y-Z AXISES HAS BEEN INTERCHANGED IN MY EXAMPLES)
My model if you would like to try out yourself -> https://drive.google.com/file/d/13QEY3arAjrYe49Wfp5P6uGv5nQrIstlM/view?usp=sharing
Thanks!  

### Version

0.0.59
